docs/context-variables.md

```diff
@@ -176,6 +176,7 @@ The `pr` context includes metadata related to the pull request.
 | `pr.approvals` | [String] | The of reviewers that approved the PR |
 | `pr.author` | String | The PR author name |
 | `pr.author_teams` | String | The teams which the PR author is member of|
+| `pr.author_is_org_member` | Bool | `true` when the PR author is a member of the org under which gitstream installed at|
```

Comment:
`gitstream` → `gitStream`

`true` if the author of the PR is a member to the organization where gitstream is installed